requirements.txt

```diff
@@ -1,49 +1,49 @@
 # requirements.txt for DefectDojo using Python 3.x
-asteval==0.9.14
+asteval==0.9.18
 bleach>=3.1.0
-celery==4.3.0
-coverage==4.5.3
+celery==4.4.0
+coverage==5.0.2
 defusedxml==0.6.0
-django_celery_results==1.1.1
-django-auditlog==0.4.5
+django_celery_results==1.1.2
+django-auditlog==0.4.7
 git+https://github.com/Maffooch/django-custom-field.git@master#egg=django-custom-field
 django-dbbackup>=3.2.0
 django-environ==0.4.5
-django-filter==2.0.0
+django-filter==2.2.0
 django-imagekit==4.0.2
-django-multiselectfield==0.1.8
+django-multiselectfield==0.1.11
 django-overextends==0.4.3
-django-polymorphic==2.0.1
-django-rest-swagger==2.1.2
-django-slack==5.13.0
+django-polymorphic==2.1.2
+django-rest-swagger==2.2.0
+django-slack==5.14.3
 django-tagging==0.4.6
 django-taggit-serializer==0.1.7
 git+https://github.com/Maffooch/django-tastypie-swagger@master#egg=django-tastypie-swagger
-django-tastypie==0.14.2
-django-watson==1.5.2
-Django==2.2.4
-djangorestframework==3.9.4
-gunicorn==19.9.0
-html2text==2018.1.9
+django-tastypie==0.14.3
+django-watson==1.5.3
+Django==2.2.9
+djangorestframework==3.11.0
+gunicorn==20.0.4
+html2text==2019.9.26
 humanize==0.5.1
 jira==2.0.0
-lxml==4.3.3
+lxml==4.4.2
 Markdown==3.1.1
-mysqlclient==1.4.2
+mysqlclient==1.4.6
 pandas>=0.24.2
 pdfkit==0.6.1
-Pillow==6.0.0  # required by django-imagekit
-psycopg2-binary==2.8.2
-cryptography==2.7
+Pillow==7.0.0  # required by django-imagekit
```

Comment:
Probably would be more conservative and use 6.2.0?

Reply:
It is working, so why should't we use the latest version?

Reply:
If you're certain it's working, then OK :-)